src/Symfony/Component/Security/Acl/Domain/Acl.php

```diff
@@ -511,7 +511,7 @@ private function insertAce($property, $index, $mask, SecurityIdentityInterface $
      */
     private function insertFieldAce($property, $index, $field, $mask, SecurityIdentityInterface $sid, $granting, $strategy = null)
     {
-        if (0 === strlen($field)) {
+        if (empty($field)) {
```

Comment:
Cause in test it will fail to assert that [`null` or `false`](http://3v4l.org/E6qS4) are empty values (was not sure we should adjust test, so I changed this to `empty()`)